nucypher/policy/conditions/types.py

```diff
@@ -39,6 +39,7 @@ class _AccessControlCondition(TypedDict):
 
 
 class RPCConditionDict(_AccessControlCondition):
+    conditionType: str
```

Comment:
Based on my previous comment about `condition_type` being part of `AccessControlCondition.Schema`, then for this `conditionType` should be moved to `_AccessControlCondition`.